src/desktop/lib/webpackPublicPath.ts

```diff
@@ -25,10 +25,11 @@ if (process.env.NODE_ENV === "production") {
   // TODO: Remove this as its temporary while routes are being converted.
   const convertedRoutes = [
     "/arist-series",
+    "/artwork",
     "/collections",
     "/collection",
     "/collect",
-    "/show/",
+    "/show",
```

Comment:
Please don't change this, it will break `shows`

Reply:
Ah, thought this was a typo